modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/query/KillSubcommand.java

```diff
@@ -54,4 +54,7 @@
 
     /** Kill snapshot operation. */
     SNAPSHOT,
+
+    /** Kill consistency tasks. */
+    CONSISTENCY,
```

Comment:
comma should be removed

Reply:
It may be removed.
As you can see it was present before my commit.
Let's keek as is (auto-extendable, without need to update old lines on extending)